Fix minor inaccuracies in RTC snapshot handling (#2116)
tinymce/tinymce-docs: _includes/configuration/rtc_snapshot.md

### `rtc_snapshot`

Real-time collaboration integrations regularly store the content, eliminating the need for a save button. The {{site.productname}} RTC plugin provides a version number to assist with storing the HTML content snapshots. These snapshots are not stored on the RTC server and must be handled by the integrator.

For any given document ID, the server guarantees the version number will only increase. It can be safely used for conflict resolution. For each document ID and version combination, the snapshot content is guaranteed to be identical.

> The snapshot callback will be executed shortly after edits are made with access to the serialized editor content. The content is retrieved through a `getContent` function to reduce CPU load if the callback decides to not use the editor content.

Reviewer: This isn't really any clearer either? Do this mean it makes a snapshot for each character, word, paragraph I add or what? I don't know enough about this to be able to make a suggestion sorry, but I think we need to be a little clearer here still.

If it's whenever a new undo level is created, maybe say that?

Reviewer 2: we don't really talk enough about undo levels...

Reviewer 3: There’s no fixed timeline for snapshots, “in response to local changes” is not just a vague documentation phrasing - it’s the reality.

It depends on how fast the local user is making changes and their lag to the server. At a technical level the snapshot event is raised any time a local change is confirmed by the server with no pending local changes to send.

Reviewer 2: ```suggestion
The snapshot callback will be executed in response to local changes, with access to the serialized editor content. The content is retrieved through a `getContent` function to reduce CPU load if the callback decides to not use the editor content.
```
?

Reviewer 2: using Andy's suggestion
